### PR TITLE
Add pages example app

### PR DESCRIPTION
**Pull request by klemola**
Howdy! 

I wrote a simple example app for showing different "pages" based on url. In practice the view is based on active page - functionality needed in many projects.

pages

Matias

### CODE REVIEW

**Comment by AdamBrodzinski** [comment]
@klemola Thank you!! 🍻 

I've been using JS SPAs for years and I struggled with the zip code routing example trying to take out the non router parts. This is just what I needed to try and get into Elm.

**Comment by klemola** [comment]
@AdamBrodzinski Happy to help!! 🍻

**Comment by domenkozar** [comment]
Posting this here for reference. I've just opened a PR to introduce navigation to my project and it's using something similar with multiple files: https://github.com/domenkozar/hydra-frontend/pull/3

File: examples/Pages.elm [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    , profilePicture = "http://placekitten.com/g/400/400"
+    }
+  ]
+
+
+initialModel : Model
+initialModel =
+  { userList = users
+  , currentPage = UserListPage
+  , selectedUserId = 0
```
Comment (cyr-l) [inline examples/Pages.elm]: @klemola This is a bit of a lie, isn't it? ;) User with `id=0` (John) is not selected yet.

Shouldn't we use a `Maybe Int` as the type?

Comment (klemola) [inline examples/Pages.elm]: @cbioley You're correct, the model is not realistic. I'll update the PR to facilitate Maybe for `selectedUser`!

Comment (klemola) [inline examples/Pages.elm]: @cbioley I've updated the PR.

Comment (cyr-l) [inline examples/Pages.elm]: @klemola Hey cool. Thanks for letting me know 👍

**Comment by klemola** [comment]
There hasn't been any input from @evancz yet. I wonder if this pull request will get merged or declined at all 🤔

**Comment by klemola** [comment]
This PR is outdated, and probably not going to be merged, so I'm closing it.

**Pull request closed without merging**